easybuild/easyconfigs/m/M4/M4-1.4.17.eb

```diff
@@ -4,19 +4,25 @@ name = 'M4'
 version = '1.4.17'
 
 homepage = 'http://www.gnu.org/software/m4/m4.html'
-description = """GNU M4 is an implementation of the traditional Unix macro processor.
- It is mostly SVR4 compatible although it has some extensions 
+
+description = """
+ GNU M4 is an implementation of the traditional Unix macro processor.
+ It is mostly SVR4 compatible although it has some extensions
  (for example, handling more than 9 positional parameters to macros).
- GNU M4 also has built-in functions for including files, running shell commands, doing arithmetic, etc.
+ GNU M4 also has built-in functions for including files, running shell
+ commands, doing arithmetic, etc.
```

Comment:
I'm OK with changing the description format, but then we should do it in *all* M4 easyconfigs...